test/integration-tests/TEST-06-SELINUX/meson.build

```diff
@@ -2,6 +2,8 @@
 
 id = find_program('id', required : true)
 uid = run_command(id, '-u', check : true).stdout().strip().to_int()
+cmd = 'if [ -e /etc/os-release ]; then . /etc/os-release; else . /usr/lib/os-release; fi; echo $ID'
```

Comment:
Hmm, I guess this checks the host os-release, rather than in the image... no?

Reply:
Yes, this won't work.

To be honest for this specific case I'm fine with adding a check in integration-test-wrapper.py that checks for TEST-06-SELINUX and specific distros from `mkosi summary` and skips if needed.